pegasus/sites.v3/code.org/public/educate/it.md

```diff
@@ -72,10 +72,9 @@ For the very best experience with all Code.org content, we recommend consulting
 | Priority                                                                                 | URL                                                                                                                                                    |
 | ---------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | **Required**                                                                             | `https://code.org`<br/>`https://*.code.org`                                                                                                            |
-| **Optional**                                                                             | `https://www.google-analytics.com`<br/>`https://www.googleoptimize.com`                                                                                |
-| **To use YouTube hosted videos** <br/>(Enhanced Privacy Mode - available late July 2018) | `https://youtube-nocookie.com/*`<br/>`https://*.googlevideo.com/*`<br/>`https://*.ytimg.com/*`                                                         |
-| **To use YouTube hosted videos** <br/>(Deprecated late July 2018)                        | `https://s.youtube.com/*`<br/>`https://www.youtube.com/*`<br/>`https://*.googlevideo.com/*`<br/>`https://*.ytimg.com/*`                                |
-| **To use Code.org hosted videos**                                                        | **Unblock:**<br/>`https://videos.code.org`<br/>**Block:**<br/>`https://www.youtube.com/favicon.ico`<br/>`https://www.youtube-nocookie.com/favicon.ico` |
+| **Optional**                                                                             | `https://cdn.optimizely.com`<br/>`https://www.google-analytics.com`<br/>`https://www.googleoptimize.com`                                               |
```

Comment:
hmm, @dju90 , actually now I am wondering if something went wrong with the merge. it looks like you may have undone what was done in https://github.com/code-dot-org/code-dot-org/pull/47056?

Reply:
I think I fixed it...how's it look now?